types/coin_regex.go

```diff
@@ -0,0 +1,972 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
This top line seems wrong? Isn't it generated by ragel?

Reply:
Ah my bad, I had this line copied over from proto-gen. Changed .